format/index.md

```diff
@@ -472,14 +488,14 @@ multiple times.
 
 As discussed above, the document **MAY** be extended to include
 meta-information as `"meta"` members in several locations: at the top-level,
-within resource objects, within link objects, and within linkage objects.
+within resource objects, within relationship objects, and within linkage objects.
```

Comment:
Anything with the word "relationship" in it misses the point for me, since I think we want to use these objects in contexts other than relationships to identify single resources. Also, I still think most people don't know what a "tuple" is and that it's kind of a scary word. (Moreover, linkage objects aren't actually tuples, since the keys aren't ordered and there can't be repeated keys.)

Maybe "resource pointer object"? Or just "resource pointer", as in "A 'resource pointer' is an object containing a "type" and "id" key, which are used to uniquely identify a resource."?